packages/test-util/src/url.spec.ts

```diff
@@ -8,4 +8,8 @@ describe('url', () => {
       )
     ).rejects.toThrowErrorMatchingSnapshot();
   });
+
+  it('[E2E] does not fail for existing link', async () => {
+    await expect(checkUrlExists('https://sap.github.io/cloud-sdk/')).resolves;
```

Comment:
I think this will always be green even if the link does not exist, please check that.